File: javascript/1015.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var xy1 = lines.shift(' ').split(' ');
var xy2 = lines.shift(' ').split(' ');
var x1 = parseFloat(xy1[0]);
var x2 = parseFloat(xy2[0]);
var y1 = parseFloat(xy1[1]);
var y2 = parseFloat(xy2[1]);

console.log(Math.sqrt(((x2 - x1) ** 2) + ((y2 - y1) ** 2)).toFixed(4))

File: python/1015.py
import math
xy1=input().split(' ')
xy2=input().split(' ')
x1=float(xy1[0])
x2=float(xy2[0])
y1=float(xy1[1])
y2=float(xy2[1])
c='{:.4f}'.format(math.sqrt(((x2 - x1) ** 2) + ((y2 - y1) ** 2)))
print(c)